### PR TITLE
Highlight incomplete block comments

### PR DESCRIPTION
**Pull request by ashkitten**

### CODE REVIEW

File: components/prism-clike.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Prism.languages.clike = {
 	'comment': [
 		{
-			pattern: /(^|[^\\])\/\*[\w\W]*?\*\//,
+			pattern: /(\/\*[\w\W]*?\*\/)|(\/\*[\w\W]*)/,
```
Comment (LeaVerou) [inline components/prism-clike.js]: Why not just make the closing bit optional then, with `(\/\*)?`? This is exactly what this does, but with more duplication.

**Comment by LeaVerou** [comment]
Thanks! See comment.

**Comment by Golmote** [comment]
I made the change manually. @LeaVerou actually just making the closing bit optional would not have worked because the lack of greediness would have stopped the highlighting far too soon. I found a way by adding a `$` anchor as an alternative to the closing bit.

**Comment by ashkitten** [comment]
Oh sorry, forgot about this. Thanks @Golmote for implementing it, when I put my new website up I'll be sure to update it with this change.

**Pull request closed without merging**